torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -43,6 +43,7 @@ def __init__(self,
         self.freeze_bn = freeze_bn if self.training else True
         self.bn = _BN_CLASS_MAP[dim](out_channels, eps, momentum, True, True)
         self.weight_fake_quant = self.qconfig.weight()
+        self.zero_bias = torch.Tensor(out_channels)
```

Comment:
Do we know that torch.Tensor always creates a tensor with zeros?. I couldnt find it in the docs

Reply:
oh..probably should be torch.zeros, torch.Tensor creates a Tensor with random values I think.